content/reference/spaceapi/yaml/sharedsecret.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: spaces.upbound.io/v1beta1
```

Comment:
i wonder what these files are - this is not a real CRD - so how you generate these files ?